src/cli/migrate/index.ts

```diff
@@ -3,7 +3,7 @@ import { log } from '../../utils/logging';
 import { nodeVersionMigration } from './nodeVersion';
 
 const migrations: Record<string, () => Promise<void>> = {
-  node20: () => nodeVersionMigration(20),
+  node22: () => nodeVersionMigration(22),
```

Comment:
I feel like it could be fine to keep 20 around just for purposes of people doing a 2-step upgrade? 🤷 maybe not